### DISCUSSION TITLE
flash wifi-adapter

### DISCUSSION THREAD

**Discussion by manfredr74** [odya/esphome-powmr-hybrid-inverter#29] [Ideas]
is it feasable to flash the wifi-adapter which has an ESP-chip herewith?

https://www.amazon.de/PowMr-POW-HVM1-5H-12V-POW-HVM2-4H-24V-POW-HVM3-6M-24V-POW-HVM6-0M-48V/dp/B09P8FVWDS

**Comment by Patogoma** [comment]
It has an ESP8266, and if you open it, there's some pins (Tx, Rx, VCC, GND, Boot) ready for program.
